src/styles/buy-field.css

```diff
@@ -0,0 +1,21 @@
+
+.buy-link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 170px;
+  height: 40px;
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: 700;
+  color: #fff;
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: none;
+}
+
+.buy-link:hover {
+  color: #00acdc;
+  background: #fff;
+  border: 1px solid #00acdc;
```

Comment:
move this into button by default

Reply:
at this moment when you hovered on your link - the border appears and adds 1px, because of box-sizing: border-box; width and height of your button is recalculating, and text could change its position, so it's better if you write border property inside the button.